libs/wire-subsystems/test/unit/Wire/UserGroupSubsystem/InterpreterSpec.hs

```diff
@@ -114,24 +115,57 @@ spec :: Spec
 spec = timeoutHook $ describe "UserGroupSubsystem.Interpreter" do
   describe "CreateGroup :: UserId -> NewUserGroup -> UserGroupSubsystem m UserGroup" $ do
     prop "team admins should be able to create and get groups" $ \team newUserGroupName ->
-      expectRight
-        . runDependencies (allUsers team) (galleyTeam team)
-        . interpretUserGroupSubsystem
-        $ do
-          let newUserGroup =
-                NewUserGroup
-                  { name = newUserGroupName,
-                    members = User.userId <$> V.fromList (allUsers team)
```

Comment:
what if the owner happens to be in the user group?  what notifications will they get then?

maybe split up the set of *all* users (including owner) randomly into two halfs, and only add the first half to the group?